### DISCUSSION TITLE
List of CSS components / classes

### DISCUSSION THREAD

**Discussion by veryKnightley** [primefaces/community#1200] [PrimeNG] [answered]
Hello all;
Its been well over a decade since I tackled any web language that wasn't filtered through an application like Wordpress.
_Horrible_, I know.

The biggest problem is the fact that everybody seems to take class names for granted.
I am refreshing my knowledge on Angular (16) with PrimeNG for the visual components, and every workshop and tutorial I find just casually drops "flex flex-column align-items-center justify-content-center" without any reference of _where_ I can find a list of all these classes, in case I want to change them to something else.

Sure, some can be intuited: justify-content-right is probably a thing; but I can either trial and error that thought or I can look it up somewhere.

Can anyone point me into the right direction of **where** that somewhere might be?
I have looked all over and somehow, Google isnt doing me much good anymore these days.

**Comment by erber014** [comment, marked as answer]
Hi, maybe https://primeflex.org/ is what you are looking for? It is a CSS library that is often used in combination with Prime component libraries.

**Reply by veryKnightley** [reply]
Yes this is perfect, thank you!
It is very nice that the website offers a clear breakdown of how all the elements are named and what they are designed to do.

**Comment by cetincakiroglu** [comment]
Hi,

In v16, our demos have been built using PrimeFlex. For detailed information, you can visit the documentation of PrimeFlex https://primeflex.org/
